Source/WebCore/rendering/MarkedText.cpp

```diff
@@ -122,6 +122,8 @@ Vector<MarkedText> MarkedText::collectForHighlights(const RenderText& renderer,
                 for (auto& rangeData : highlightRegister->map().get(highlightName)->rangesData()) {
                     if (!highlightData.setRenderRange(rangeData))
                         continue;
+                    if (auto* staticRange = dynamicDowncast<StaticRange>(rangeData->range()); staticRange && !staticRange->isValid())
```

Comment:
should this be
```suggestion
                    if (auto* staticRange = dynamicDowncast<StaticRange>(rangeData->range()); !staticRange || !staticRange->isValid())
```

(ie, should it `continue` if the range is not a StaticRange?)

Reply:
It should not `continue` if range is not static range bc we support live ranges too. I think you're suggesting to `continue` if it's not static range?

Reply:
ah I see, just wanted to confirm!